applications/datsphere.md

```diff
@@ -0,0 +1,96 @@
+# Crust Grant Proposal
+
+* **Project Name:** DatSphere  
+* **Team Name:** DatSphere  
+* **Payment Address:** 0xdd2f8a0e413ddd58bf60a1c9218723f312671e33  
+* **Receivable Token:** USDT ERC20  
+
+### Overview
+Please provide the following:
+  * A project name: DatSphere  
+  * A brief description of your project: An open-source, decentralized storage service built on dat, IPFS, Crust.  
+  * An indication of how your project relates to / integrates into Crust: We are seeing ourselves as eco-partner, We regard decentralized storage as core value, would like to provide handy service to users. 
+  * An indication of why your team is interested in creating this project: We've always been interested in decentralized storage service, the advent of dat, IPFS, Crust, let us realize that we can develop something special service.
+
+### Project Details 
+
+We expect the teams to already have a solid idea about your project's expected final state. Therefore, we ask the teams to submit (with urls or IPFS file CIDs included) at following :
+
+* Mockups/designs of any UI components
+* Data models / API specifications of the core functionality.
+* An overview of the technology stack to be used: Egg, Vue, dat, Hypercore Protocol, ipfs
+* Documentation of core components, protocols, architecture, etc. to be deployed: Dat/IPFS/Crust White Paper
+* PoC/MVP or other relevant prior work or research on the topic
+
+### Ecosystem Fit 
+
+Help us locate your project in the Crust landscape and what problems it tries to solve by answering each of these questions:
+
+* Where and how does your project fit into the ecosystem? We provide centralized personal storage service on top of Crust. Which will lower the barrier to entry. We also provide handlful toolkit help user set up their own service.
+* What need(s) does your project meet? : User can easily set up their own decentralized storage service.
+* Are there any other projects similar to yours in the IPFS/Crust ecosystem? 
+  No. There is a service called Crato, which implemented IPFS pin service to Crust. But this is a semi-finished product, and not for personal, not using dat tech stack either.
```

Comment:
The dat protocol is not currently supported by Crust. How do you provide, how do you provide dat services based on Crust?

Reply:
Yes, it is.
I checked Crust-cli, Crust-js, do not support protocol other than IPFS.
So I will integrate Crust with IPFS firstly.
If Crust team has plans to support dat, I will integrate it in the future.